### ISSUE TITLE
wrong integral value

### ISSUE THREAD

**Issue by nschloe** [sympy/sympy#19427]
sympy reports 0 on an integral that isn't 0.
```python
from sympy import integrate, Symbol, sqrt

x = Symbol("x")

print(integrate((x ** 4) * sqrt(1 - x ** 2), (x, -1, 1)))
print(integrate((-2 * x ** 2) * sqrt(1 - x ** 2), (x, -1, 1)))
print(integrate((1) * sqrt(1 - x ** 2), (x, -1, 1)))

# sum
print(integrate((x ** 4 - 2 * x ** 2 + 1) * sqrt(1 - x ** 2), (x, -1, 1)))
```
```
pi/16
-pi/4
pi/2
0
```
The last line should be the sum of the previous lines, `5/16 pi`.

**Comment by stevenTan01** [comment]
Hi, I'm new to open source contributing and would like to take a look and see if I can fix this issue.

**Comment by nschloe** [comment]
Confirmed.

**Comment by oscarbenjamin** [comment]
I think this requires more investigation or at least a test should be added.

If it is fixed then we need to identify what fixed it and we need to know if the underlying bug is fixed or just avoided.

If we are clear that the underlying bug is fixed then this makes a good case for a regression test that should be added to the test suite.

**Comment by oscarbenjamin** [comment]
Also these integrals should not be so slow. I'm not sure what causes that but further investigation would be good.

**Comment by oscarbenjamin** [comment]
I bisected the fix to 45e198080dbc9d053d4d1bb6e156f8bdab578035 from #22052

**Comment by oscarbenjamin** [comment]
So I think that the bug is fixed.

In that case I think it would be good to add the 4 integrals from the OP as regression tests and then this can be closed.
